ui/js/directives/treeherder/bottom_nav_panel.js

```diff
@@ -34,67 +34,3 @@ treeherder.directive('thRelatedBugQueued', function () {
         templateUrl: 'partials/main/thRelatedBugQueued.html'
     };
 });
-
-treeherder.directive('thFailureClassification', [
-    'thClassificationTypes',
-    function (thClassificationTypes) {
-        return {
-            scope: {
-                failureId: "=",
-                jobResult: "="
-            },
-            link: function (scope) {
-                scope.$watch('[failureId, jobResult]', function () {
-                    if (scope.failureId) {
-                        scope.classification = thClassificationTypes.classifications[scope.failureId];
-                        scope.hoverText=scope.classification.name;
-                        scope.iconCls = (scope.failureId === 7 ? "fa-star-o" : "fa fa-star") +
-                                         " star-" + scope.jobResult;
-                    }
-                });
-            },
-            template: '<span title="{{hoverText}}">' +
-                '<i class="fa {{iconCls}}"></i>' +
-                '</span> {{hoverText}}'
-        };
-    }]);
-
-treeherder.directive('thSimilarJobs', [
```

Comment:
This was removed in 0163860eb4713f58d91f86860674b627acc17eee but I'm guessing accidentally re-added when this other change was rebased: https://github.com/mozilla/treeherder/commit/d6152133#diff-219b9ee5371f4e7ba37de132ed0e371aR126